src/script/lua_api/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(common_SCRIPT_LUA_API_SRCS
 	${CMAKE_CURRENT_SOURCE_DIR}/l_areastore.cpp
+	${CMAKE_CURRENT_SOURCE_DIR}/l_auth.cpp
```

Comment:
Also add to https://github.com/minetest/minetest/blob/master/build/android/jni/Android.mk

Reply:
Done, but I have not been able to successfully build on Android yet, so I can't test this.

I see that not all unit test files are listed. Is this on purpose?

Reply:
Probably it compiled fine on Android so nobody noticed that they were forgotten - or it's on purpose and the documentation is missing. :man_shrugging: